pkg/services/database/database.go

```diff
@@ -243,3 +243,23 @@ func (d *database) findPreloadWitLimitAndPageQuery(
 
 	return d.connectionRead.Table(table).Where(where).Limit(limit).Offset(page * limit)
 }
+
+// Deprecated: RawQuery starts a transaction and try to execute the raw query into database.
+// is not recommended using this and the method will not be available after cli v2.10.0
+func (d *database) RawQuery(rawQuery string) error {
+	sqlDB, err := d.connectionWrite.DB()
+	if err != nil {
+		return err
+	}
+
+	tx, err := sqlDB.Begin()
+	if err != nil {
+		return err
+	}
```

Comment:
Why you can use `StartTransaction` method?

Reply:
we cannot access the pure sql object with StartTransaction, to be able to execute a query with more than one statement i couldn't find another way, i welocme suggestions

Reply:
okay, we can keep in this format